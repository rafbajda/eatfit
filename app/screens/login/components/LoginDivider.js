import React from 'react';
import Divider from 'react-native-divider';
import { DividerContainer, DividerText } from '../styles/loginDividerStyles';
import { lightGrey } from '../../../shared/constants/colors';

const LoginDivider = () => {
    return (
        <DividerContainer>
            <Divider borderColor={lightGrey} orientation="center">
                <DividerText>OR</DividerText>
            </Divider>
        </DividerContainer>
    );
};

export default LoginDivider;
