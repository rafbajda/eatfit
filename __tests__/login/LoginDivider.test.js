import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import NavigationTestUtils from 'react-navigation/NavigationTestUtils';
import LoginDivider from "../../app/screens/login/components/LoginDivider";

describe('LoginDivider snapshot', () => {
    jest.useFakeTimers();
    beforeEach(() => {
        NavigationTestUtils.resetInternalState();
    });

    it('renders component', async () => {
        const tree = renderer.create(<LoginDivider />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
