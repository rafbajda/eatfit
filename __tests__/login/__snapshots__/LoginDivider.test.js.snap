// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginDivider snapshot renders component 1`] = `
<RCTScrollView
  automaticallyAdjustContentInsets={false}
  className="sc-bdVaJa gLzyJe"
  contentContainerStyle={
    Array [
      Object {
        "padding": undefined,
      },
      undefined,
    ]
  }
  contentInset={
    Object {
      "bottom": 0,
    }
  }
  enableAutomaticScroll={true}
  enableOnAndroid={false}
  enableResetScrollToCoords={true}
  extraHeight={75}
  extraScrollHeight={0}
  getScrollResponder={[Function]}
  handleOnScroll={[Function]}
  keyboardDismissMode="interactive"
  keyboardOpeningTime={250}
  keyboardShouldPersistTaps="handled"
  keyboardSpace={0}
  onScroll={[Function]}
  resetKeyboardSpace={[Function]}
  resetScrollToCoords={
    Object {
      "x": 0,
      "y": 0,
    }
  }
  scrollEventThrottle={1}
  scrollForExtraHeightOnAndroid={[Function]}
  scrollIntoView={[Function]}
  scrollToEnd={[Function]}
  scrollToFocusedInput={[Function]}
  scrollToPosition={[Function]}
  showsVerticalScrollIndicator={true}
  style={
    Object {
      "backgroundColor": "transparent",
      "flex": 1,
    }
  }
  update={[Function]}
  viewIsInsideTabBar={false}
>
  <View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 24,
          "marginVertical": 6,
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "borderBottomWidth": 1,
              "height": 24,
              "transform": Array [
                Object {
                  "translateY": -12,
                },
              ],
            },
            Object {
              "borderColor": "#dfe3ee",
            },
            false,
            Object {
              "flex": 1,
            },
          ]
        }
      />
      <Text
        style={
          Array [
            Object {
              "fontSize": 16,
              "fontWeight": "500",
              "paddingHorizontal": 24,
            },
            Object {
              "color": "rgba(0,0,0,.85)",
            },
          ]
        }
      >
        <Text
          className="sc-bwzfXH cwBKGu"
          style={
            Object {
              "color": "#000",
              "fontFamily": "System",
              "fontSize": 16,
            }
          }
          uppercase={false}
        >
          [missing "en.dividers.or" translation]
        </Text>
      </Text>
      <View
        style={
          Array [
            Object {
              "borderBottomWidth": 1,
              "height": 24,
              "transform": Array [
                Object {
                  "translateY": -12,
                },
              ],
            },
            Object {
              "borderColor": "#dfe3ee",
            },
            false,
            Object {
              "flex": 1,
            },
          ]
        }
      />
    </View>
  </View>
</RCTScrollView>
`;
